config/locales/views/about/en.yml

```diff
@@ -9,6 +9,12 @@ en:
           CircuitVerse is an open-source project with an active community. Checkout the contribute page for more detail."
     terms_of_service: "Terms of Service"
     privacy_policy: "Privacy Policy"
+    core_team_heading: "CORE TEAM"
+    core_team_description: "CircuitVerse is lead by an experienced team that is dedicated to innovation and excellence."
+    mentors_team_heading: "MENTORS"
+    mentors_team_description: "Our mentors supports and encourages the professional development of the mentee and provides active guidance to help them for the fruitful development of our community."
```

Comment:
A small nitpick. 
"Our mentors support and encourage the professional development of the mentee and provide active guidance to help them for the fruitful development of our community." sounds more gramatically correct
Everything else looks great!